src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -174,11 +169,16 @@
             },
             selectedAreaIds() {
                 const selectedAreaIdSet = flattenToIdSet(this.selectedAreaFilterOptions.map(o => o.id), this.flattenedAreas);
+
+                //Should also ensure include top level (country) included if no filters selected
+                const selectedOptions = this.selections.selectedFilterOptions[this.areaFilterId];
+                if (!selectedOptions || selectedOptions.length == 0) {
```

Comment:
This fixes a previously undiagnosed bug which meant that data at level 0 (country) was not shown when selected because it wasn't included in the selectedAreaIds set